packages/components/src/components/post-accordion/post-accordion.tsx

```diff
@@ -25,33 +25,33 @@ export class PostAccordion {
   @Prop() readonly headingLevel!: HeadingLevel;
 
   @Watch('headingLevel')
-  validateHeadingLevel(newValue = this.headingLevel) {
-    if (!newValue) return;
-    checkOneOf(
-      this,
-      'headingLevel',
-      HEADING_LEVELS,
-      'The `heading-level` property of the `post-accordion` must be a number between 1 and 6.',
-    );
-
-    this.accordionItems.forEach(item => {
-      item.setAttribute('heading-level', String(newValue));
-    });
+  validateHeadingLevel() {
+    if (!checkNonEmpty(this, 'headingLevel')) {
```

Comment:
There is a convention for functions that return a boolean: https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig Can you try and apply it here? 

Otherwise would you be able to know if the property is required or not by inspecting the `this`? If so the check empty/non empty could be done automatically in the checkers and not manually in the components.